Pipfile

```diff
@@ -17,6 +17,7 @@ pandas = "*"
 xlrd = "*"
 pyyaml = "*"
 deepdiff = "*"
+flask-deprecate = "*"
```

Comment:
I think this also needs to get added to `Pipfile.lock`?

Reply:
Ah, for some reason I thought this got .gitignored (like a `node_modules` directory). I'll re-push with those changes.